arch/ceva/src/common/up_assert.c

```diff
@@ -296,7 +296,7 @@ static void _up_assert(int errorcode)
 
   /* Are we in an interrupt handler or the idle task? */
 
-  if (CURRENT_REGS || running_task()->pid == 0)
+  if (up_interrupt_context() || sched_idletask())
```

Comment:
@xiaoxiang781216 please review this place. Change seems to be equivalent, but maybe there is something that I miss.